test/library/standard/Set/setOperatorOrderTest.chpl

```diff
@@ -37,12 +37,23 @@ s2.add(b);
 s1.add(c);
 s2.add(d);
 
-writeln(s1 | s2);
-writeln(s1 + s2);
-writeln(s1 - s2);
-writeln(s1 & s2);
-writeln(s1 ^ s2);
+printSortedSet(s1 | s2);
+printSortedSet(s1 + s2);
+printSortedSet(s1 - s2);
+printSortedSet(s1 & s2);
+printSortedSet(s1 ^ s2);
 writeln(s1 <= s2);
 writeln(s2 <= s1);
 writeln(s1 >= s2);
 writeln(s2 >= s1);
+
+proc printSortedSet(s) {
```

Comment:
It looks like `set` has `toArray()`, so you could just do `writeln(s.toArray().sorted());`

Reply:
Thanks for pointing that out, that makes much more sense...